index.ts

```diff
@@ -138,4 +138,32 @@ export class Test {
     stack.trim();
     Test.assert(msg, x !== y, `Expected ${x} !== ${y}\n${stack}`);
   }
+
+  // assert that `x` is greater than `y`, naming the assertion with `msg`, which will be used as a label in the CLI runner.
+  static assertGreaterThan(msg: string, x: any, y: any) {
+    const stack = new Error().stack || "";
+    stack.trim();
+    Test.assert(msg, x > y, `Expected ${x} > ${y}\n${stack}`);
+  }
+
+  // assert that `x` is greater than or equal to `y`, naming the assertion with `msg`, which will be used as a label in the CLI runner.
+  static assertGreaterThanOrEqualTo(msg: string, x: any, y: any) {
+    const stack = new Error().stack || "";
+    stack.trim();
+    Test.assert(msg, x >= y, `Expected ${x} >= ${y}\n${stack}`);
+  }
+
+  // assert that `x` is less than `y`, naming the assertion with `msg`, which will be used as a label in the CLI runner.
+  static assertLessThan(msg: string, x: any, y: any) {
+    const stack = new Error().stack || "";
+    stack.trim();
+    Test.assert(msg, x < y, `Expected ${x} < ${y}\n${stack}`);
+  }
+
+  // assert that `x` is less than or equal to `y`, naming the assertion with `msg`, which will be used as a label in the CLI runner.
+  static assertLessThanOrEqualTo(msg: string, x: any, y: any) {
+    const stack = new Error().stack || "";
+    stack.trim();
+    Test.assert(msg, x <= y, `Expected ${x} <= ${y}\n${stack}`);
+  }
```

Comment:
I wonder if we should converge on [`node:assert` types](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/assert.d.ts) for test assertions? It's pretty well-supported across platforms and browsers.

Reply:
@chrisdickinson probably a good idea! I think as long as they still wrap our own internal functions so the outputs are consistent across test runs, making the most idiomatic API for a language ecosystem is the right move. 

I will merge as-is, and we can re-visit (perhaps when/if you'd like to take it on... :smile: )